py/DREAM/Settings/Equations/DistributionFunction.py

```diff
@@ -338,6 +356,11 @@ def todict(self):
             data['transport'] = self.transport.todict()
             data['fullIonJacobian'] = self.fullIonJacobian
 
+        if self.mode != DISTRIBUTION_MODE_NUMERICAL:
```

Comment:
Why not ``self.mode == DISTRIBUTION_MODE_ANALYTICAL``?

Reply:
The intent was to later have multiple analytic modes (with/without contributions to n_hot and j_hot for example)